### ISSUE TITLE
just lost my Job because of this package

### ISSUE THREAD

**Issue by Root-Control** [hiro1107/nestjs-supabase-auth#10]
Terrible validation, I had to create different patches for this, but the decorators are not validating anything, If my boss doesn't suggest it, I don't use it, as my plan was to use the base jwt extractor that comes with Nestjs, thanks.

**Comment by tgirotto** [comment]
... good for you I guess?

**Comment by Root-Control** [comment]
> ... good for you I guess?

100%, that client was a Shit.

**Comment by Sefton419** [comment]
@tgirotto how has your experience been using this?

**Comment by tgirotto** [comment]
@tgirotto I had to fork it at https://github.com/tgirotto/nestjs-supabase-auth-v2 because this package did not support the new supabase versions. We use it in production and it seems fine.

**Comment by homj** [comment]
hey @tgirotto thanks for that fork. Did you recently delist it form npmjs? My Github Action fails when trying to install `nestjs-supabase-auth-v2`

**Comment by tgirotto** [comment]
@homj yeah I had to delist yesterday because of an issue with my account. Let me know if there is something I can do to help, but it sounds like publishing this repo again from someone else's account could be the fastest way to get things to work again (don't have much bandwidth at the moment)..

**Comment by Root-Control** [comment]
> hey [@tgirotto](https://github.com/tgirotto) thanks for that fork. Did you recently delist it form npmjs? My Github Action fails when trying to install `nestjs-supabase-auth-v2`

Bro, honestly, using this package or creating a new one, I prefer to create a new one using Nestjs Modules + ChatGPT , Cursor, Gemini or others, it is more easy to do
